lib/osf-components/addon/components/open-resource/template.hbs

```diff
@@ -0,0 +1,23 @@
+<p local-class='OpenResource_item'>
+    <OsfLink
+        data-analytics-name='Badge link to registration'
+        data-test-registration-link='{{@registration}}'
+        @route='registries.overview.resources'
+        @models={{@registration.id}}
+    >
+        <span
+            data-test-badge-icon={{@resourceType}}
+        >
+            <label data-test-resource-badge local-class='OutputResource__label'>
+                <img
+                    data-analytics-name={{@resourceType}}
+                    data-test-badge-icon
+                    src={{this.getSrc}}
+                    local-class='OutputResource_img'
+                    alt={{t (concat 'registries.overview.resources.' @resourceType '.alt_text')}}
```

Comment:
Why not use the alt text that I used on my component? It was more descriptive for people who can't see the icon, since it said what state the icons were in.

Reply:
Brian, is this the one?

<img width="589" alt="image" src="https://user-images.githubusercontent.com/82047646/183430606-dc1c49be-b039-49db-adb5-da5b824c4006.png">